maxim/tests/test_groq.py

```
@@ -270,6 +585,52 @@ def test_generation_name_header(self):
+                model="llama-3.1-8b-instant",
+                messages=[{"role": "user", "content": "What's the weather like in New York?"}],
+                max_tokens=150,
+                temperature=0.7,
+                tools=tools,
+                tool_choice="auto",  # Let the model decide whether to use tools
+            )
+            
+            print(response)
+            
+            # Verify response structure
+            self.assertIsNotNone(response)
```

Ruff (0.12.2)

Use a regular `assert` instead of unittest-style `assertIsNotNone`

Replace `assertIsNotNone(...)` with `assert ...`

(PT009)